src/pages/Search/SyncErrors.tsx

```diff
@@ -107,12 +120,64 @@ const SyncErrors = ({
                         }
                     }}
                 >
-                    Retry synchronization
+                    {resyncStarted ? (
+                        <>
+                            Retry synchronization
+                            <Progress.Circular size={16} />
+                        </>
+                    ) : (
+                        <div>Retry synchronization</div>
+                    )}
                 </Button>
-                <Button onClick={deleteFailedUpdates}>
-                    Delete failed updates
+                <Button
+                    onClick={(): void => {
+                        setShowDeleteConfirmation(true);
+                    }}
+                >
+                    Delete failed offline changes
                 </Button>
             </ButtonWrapper>
+
+            {showDeleteConfirmation == true ? (
+                <Scrim
+                    isDismissable
+                    onClose={(): void => {
+                        setShowDeleteConfirmation(false);
```

Comment:
Might need to set isSure to false here (to avoid the checkbox being checked after re-opening the dialog). We'll probably need to do this for the dialog in the bookmarks page too